ntttcp/ntttcp.Config.json

```diff
@@ -0,0 +1,77 @@
+{
+    "NtttcpDefault": {
+        "Iterations"  : 5,
+        "StartPort"   : 50002,
+        "tcp" : {
+            "BufferLen"   : [65536],
+            "Connections" : [1, 64],
+            "Options"     : ""
+        },
+        "udp" : {
+            "BufferLen"  : [1472],
+            "Connections": [1, 64],
+            "Options"    : ""
+        },
+        "Warmup"      : 2,
+        "Cooldown"    : 2,
+        "Runtime"        : 60,
+        "RecvOptions" : "",
+        "SendOptions" : ""
+    },
+
+    "NtttcpAzure": {
+        "Iterations"  : 6,
+        "StartPort"   : 50002,
+        "tcp" : {
+            "BufferLen"   : [65536],
+            "Connections" : [1, 64, 256],
+            "Options"     : ""
+        },
+        "udp" : {
+            "BufferLen"   : [1472],
+            "Connections" : [1, 64, 128],
+            "Options"     : ""
+        },
+        "Warmup"      : 2,
+        "Cooldown"    : 2,
+        "Runtime"        : 60,
+        "RecvOptions" : "",
+        "SendOptions" : ""
+    },
+
+    "NtttcpDetail": {
+        "Iterations"  : 6,

```

Comment:
Might be way too much. Should we stick to the same # of iterations as the windows NPT runs? 
Detail runs are typically quick sampling tests. so the goal here is more combinations and fewer iterations. Runtime could be small too